util/src/main/scala/org/ensime/util/path.scala

```diff
@@ -0,0 +1,29 @@
+package org.ensime.util
+
+import java.net.{ JarURLConnection, URL }
+import java.nio.charset.Charset
+import java.nio.file.{ Path => JPath }
+
+import com.google.common.io.Files
+/**
+ * Created by harsh on 15/06/16.
```

Comment:
we try not to have these sorts of comments. Note that you'll need to run the sbt-header parser to add copyright/license info to the top of the file. See `.drone.yml` for the command.